platform/mds/mds/src/test/java/org/motechproject/mds/it/osgi/MdsBundleIT.java

```diff
@@ -168,14 +183,45 @@ public void tearDown() throws Exception {
 
     @Test
     public void testEntitiesBundleInstallsProperly() throws Exception {
+
+        //Run tests with l2 cache active
+        testEntitiesBundleInstallsProperly(true);
+
+        //Turn off l2 cache
+        outputStream =  new FileOutputStream(configurationFile);
+        Properties datanucleusProps = coreConfigurationService.loadDatanucleusConfig();
+        datanucleusProps.setProperty("datanucleus.cache.level2.type", "none");
+        datanucleusProps.store(outputStream, null);
+        outputStream.close();
+
+        //Run tests without l2 cache
+        testEntitiesBundleInstallsProperly(false);
+
+        //Remove added l2 cache property after test was done
+        //New FileOutputStream is created so the config won't be appended
+        //to existing one
+        outputStream = new FileOutputStream(configurationFile);
+        datanucleusProps.remove("datanucleus.cache.level2.type", "none");
```

Comment:
what if it was none to being with? This is academical, but would be better to store the value of the original property. Moreover this whole operation should be performed in teardown, currently it will only revert the setting if the test succeeds